hunting/linux/queries/command_and_control_via_unusual_file_downloads_from_source_addresses.toml

```
@@ -0,0 +1,29 @@
+[hunt]
+author = "Elastic"
+description = """
+This hunt identifies unusual file download activities on Linux systems. It detects instances where commonly used download utilities such as curl and wget are executed with command lines that contain IP addresses, which can indicate potentially suspicious file downloads.
+"""
+integration = ["endpoint"]
+uuid = "0d061fad-cf35-43a6-b9b7-986c348bf182"
+name = "Unusual File Downloads from Source Addresses"
+language = ["ES|QL"]
+license = "Elastic License v2"
+notes = [
+    "Detects instances where download utilities like curl and wget are used with IP addresses in their command lines.",
+    "Monitors for potentially suspicious file downloads, which are often seen in malicious activities.",
+    "Uses process command line counting in conjunction with host counting to minimize false positives caused by legitimate downloads.",
+    "The process command line count threshold is set to <= 10, and the host count threshold is set to <= 5 to balance detection and noise."
+]
+mitre = ["T1071.001", "T1071.004"]
+
+query = [
+'''
+from logs-endpoint.events.process-*
+| where @timestamp > now() - 7 day
+| where host.os.type == "linux" and event.type == "start" and process.name in ("curl", "wget") and process.command_line rlike """.*[0-9]{1,3}(\.[0-9]{1,3}){3}.*"""
+| stats cc = count(), host_count = count(host.name) by process.command_line, process.executable, host.name
```

count(host.name) by host.name ? 

```suggestion
| stats cc = count(), host_count = count_distinct(host.id) by process.command_line, process.executable
```